gluten-core/src/main/scala/io/glutenproject/execution/BasicScanExecTransformer.scala

```diff
@@ -81,6 +79,7 @@ trait BasicScanExecTransformer extends TransformSupport {
       case byte: ByteType =>
       case array: ArrayType =>
       case bool: BooleanType =>
+      case bool: MapType =>
```

Comment:
Maybe `case map: MapType` is better here?

Reply:
Updated.